app/com/gu/memsub/auth/common/KeystoreCredentials.scala

```diff
@@ -0,0 +1,7 @@
+package com.gu.memsub.auth.common
```

Comment:
I've just copied these from this library to avoid horrible dependency annoyance, please let me know your opinion on keeping it separate or not.
https://github.com/guardian/memsub-common-play-auth

Reply:
We could pull this into membership-common instead, but I don't have a particularly strong view either way here...

Reply:
Perhaps @davidfurey has a view, as presumably we'll run into similar issues when we do the membership-frontend upgrade?

Reply:
decided not to pursue this, as it will add more dependencies to membership common.  More code I mind less than more dependencies.